components/esp_modem/include/cxx_include/esp_modem_exception.hpp

```diff
@@ -40,27 +46,33 @@ class esp_err_exception: virtual public std::exception {
     std::string message;
 };
 #else
-#define THROW(exception) abort()
+#define ESP_MODEM_THROW(exception) abort()
```

Comment:
Add a function call to give user information before calling abort.

Reply:
Make sense! Have added a stub implementation of `class esp_err_exception` that prints the details before `abort();`